api/src/comment/dto/comment.dto.ts

```diff
@@ -22,3 +22,17 @@ export class CommentDto {
   @IsDate()
   updatedAt: Date;
 }
+
+export class CommentQueryDto {
+  @Expose()
+  @IsString()
+  commentId?: string;
```

Comment:
I think you can use `IsOptional()` as well since you're marking them as optional fields, same for the two below